src/software/proto/message_translation/proto_creator_primitive_visitor.h

```diff
@@ -31,11 +31,11 @@ class ProtoCreatorPrimitiveVisitor : public PrimitiveVisitor
     void visit(const StopPrimitive &stop_primitive) override;
 
     /**
-     * Create a PrimitiveMsg from the given Primitive
-     * @param primitive The primitive to create the PrimitiveMsg from
-     * @return A PrimitiveMsg representation of the given primitive
+     * Create a TbotsProto::Primitive from the given Primitive
+     * @param primitive The primitive to create the TbotsProto::Primitive from
+     * @return A TbotsProto::Primitive representation of the given primitive
```

Comment:
Whitespace between description, param, and return.

Reply:
fixed